site/content/docs/user/configuration.md

```diff
@@ -102,7 +102,7 @@ Multiple details of the cluster's networking can be customized under the
 
 #### IP Family
 
-KIND has support for IPv4, IPv6 and dual-stack clusters, you can switch from the default of IPv4 by setting:
```

Comment:
This "you can switch from the default of IPv4 by setting:" felt strange. When I read it it felt like a config option was missing after the colon, but I see everything is described in the text below.
This attempts to fix that problem, but It feels un-elegant to me.

Reply:
we need english native people, @BenTheElder , we still need to mention that the default is IPv4 in the new sentence

Reply:
suggested something below, I agree that this is awkward currently